scipy/stats/__init__.py

```diff
@@ -259,6 +259,8 @@
    ks_2samp
    epps_singleton_2samp
    mannwhitneyu
+   mann_whitney_u
+   mannwhitneyu2
```

Comment:
Not all of these functions will exist. The contents of `mannwhitneyu2` will replace those of `mannwhitneyu` or `mann_whitney_u`.